protocol/0086-ASPR-activity_streak_program.md

```diff
@@ -0,0 +1,126 @@
+# Activity Streak Program
+
+The purpose of the activity streak program is to reward loyal, active traders with the following tiered benefits:
+
+- a greater share of rewards schemes
+- an accelerated vesting rate on locked rewards
+
+Parties access higher tiers and greater benefits by maintaining an activity streak. The length of a streak is measured in epochs and a party is considered active if they made a trade or held an open position during the epoch. If a party is inactive for more than a specified number of epochs they lose their streak.
+
+## Network parameters
+
+- `rewards.activityStreak.benefitTiers`: is an ordered list of dictionaries defining the requirements and benefits for each tier.
+- `rewards.activityStreak.inactivityLimit`: the maximum number of epochs a trader can be inactive before loosing their streak.
```

Comment:
this doesn't seems to be used anywhere in the spec?

Reply:
Yeah good catch, it used in the section about setting the streak length but under a different name.. updated it so it uses `inactivityLimit`.